src/index.js

```diff
@@ -5,13 +5,13 @@ import Config from "./config";
 import {getUserFromJwt} from "./lib/identityActions";
 import "./index.css";
 import registerServiceWorker from "./registerServiceWorker";
-import OneGraphApolloClient from "onegraph-apollo-client";
```

Comment:
OneGraphApollo Client uses AuthGuardian library, which means it may need to stay.

Reply:
I think I am wrong about this one. Disregard.